### PR TITLE
doc/compiler.md:fix taproot_compiler.pdf link

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

File: doc/compiler.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ to re-arrange the policy tree to produce a even more efficient Miniscript. When
...
 
 ### Non-determinism and stability guarantees of compiler
 
 The compiler outputs are not stable. They can change from version to version, machine to machine or even execution to execution on the same machine. The rust and C++ versions can produce different outputs even if the policy is the same. There could also be other implementations of compiler optimizing for different resource limits.
-However, the compiler will **always** output a valid Miniscript which might not be the same as some previous execution. As a simple example, `and_b(A,B)` could become `and_b(B,A)`. Therefore, it is **not recommended** to use policy as a stable identifier for a Miniscript. You should use the policy compiler once, and then use the Miniscript output as a stable identifier.
\ No newline at end of file
+However, the compiler will **always** output a valid Miniscript which might not be the same as some previous execution. As a simple example, `and_b(A,B)` could become `and_b(B,A)`. Therefore, it is **not recommended** to use policy as a stable identifier for a Miniscript. You should use the policy compiler once, and then use the Miniscript output as a stable identifier.
```
Comment (tcharding) [inline doc/compiler.md:40]: Why is this here? I think it can be removed.

**Review by tcharding — APPROVED** [review]
ACK eb10eb2c329cba5d3a2e16dca2c09c1e853d8ef6

**Review by apoelstra — APPROVED** [review]
ACK eb10eb2c329cba5d3a2e16dca2c09c1e853d8ef6
